torch/csrc/distributed/rpc/init.cpp

```diff
@@ -42,8 +42,10 @@ PyObject* rpc_init(PyObject* /* unused */) {
           .def(
               "join", &RpcAgent::join, py::call_guard<py::gil_scoped_release>())
           .def(
-              "sync",
-              &RpcAgent::sync,
+              "sync", &RpcAgent::sync, py::call_guard<py::gil_scoped_release>())
+          .def(
+              "_get_rpc_timeout",
```

Comment:
Question, if the RpcAgent will handle timeout internally, why do we need to expose this API in Python and C++? Is this only for test and debugging?

Reply:
I think what @rohan-varma mentioned was since we allow users to set timeout, we need to have a getter to allow users to inspect this value as well.

Reply:
I see, makes sense